bin/animation_assets/manifest_builder.rb

```diff
@@ -436,6 +441,11 @@ def generate_json(animation_metadata, alias_map, category_map)
     )
   end
 
+  def generate_spritelab_manifest_filename(locale = nil)
+    filename = locale ? `spritelabCostumeLibrary.#{locale}.json` : 'spritelabCostumeLibrary.json'
```

Comment:
I don't think we want to use backquote. This will execute as a shell command. It should probably be double quotes?
```suggestion
    filename = locale ? "spritelabCostumeLibrary.#{locale}.json" : 'spritelabCostumeLibrary.json'
```

Reply:
Removed! Thanks for pointing this out!